applications/openshift/scheduler/scheduler_address_argument/oval/shared.xml

```diff
@@ -0,0 +1,30 @@
+<def-group>
+  <definition class="compliance" id="scheduler_address_argument" version="1">
+    <metadata>
+      <title>Configure the Scheduler Service Network Address</title>
+      <affected family="unix">
+        <platform>multi_platform_ocp</platform>
+      </affected>
+      <description>Scheduler network address should be configured to the loopback address.</description>
+    </metadata>
+    <criteria operator="AND">
+      <extend_definition comment="Runtime configuration is correct" definition_ref="ocp_service_runtime_config_scheduler_address_argument" />
+      <criterion comment="scheduler network address is configured" test_ref="test_scheduler_address_argument" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="scheduler network address is configured" id="test_scheduler_address_argument" version="1">
+    <ind:object object_ref="object_scheduler_address_argument" />
+    <ind:state state_ref="state_scheduler_address_argument" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_scheduler_address_argument" version="1">
+    <ind:filepath>/etc/origin/master/master-config.yaml</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*kubernetesMasterConfig\:(?:[^\n]*\n+)+?[\s]*schedulerArguments\:(?:[^\n]*\n+)+?[\s]*address\:[\s]*[\n]+[\s]*-[\s]+(\S+)[\s]*$</ind:pattern>
```

Comment:
Have seen the configs expressed multiple ways:

`````
kubernetesMasterConfig:
  schedulerArguments:
    address: 
      - 'value here'
`````

and
`````
kubernetesMasterConfig:
  schedulerArguments:
    address: 
      - noticks
`````

and
`````
kubernetesMasterConfig:
  schedulerArguments:
    address: theValue
`````

and
`````
kubernetesMasterConfig:
  schedulerArguments:
    address: 'the value'
`````

The regex doesn't seem to account for all of them. For example when using 
`````
kubernetesMasterConfig:
  schedulerArguments:
    address: 
      - 'value'
`````

the capture group ``(\S+)`` will include the ticks ``'``